### PR TITLE
Initial implementation for force-unlock

### PR DESCRIPTION
**Pull request by favoretti**
Sometimes ADO pipeline cancels the runs mid-run leaving stale locks hanging. This would allow people to create separate pipelines to unlock those cases.

### CODE REVIEW

**Comment by favoretti** [comment]
@charleszipp Long time no speak. What do you think of this one?

**Review by charleszipp — CHANGES_REQUESTED** [review]
Thanks for this addition @favoretti ! I have experienced this occasionally as well so think this will be great addition.

Would you mind adding a test to ensure the command is being issued appropriately? Im thinking at minimum we need one test to ensure the handler issues the command as terraform expects. 

I will check in on the pipeline to make sure there are no other issues.

**Comment by charleszipp** [comment]
@favoretti also would you mind updating the overview.md at the root to mention the support for this new command? I think just adding it to the list of supported commands should be good enough as there isnt much explanation needed about how to use this.

image

**Comment by favoretti** [comment]
@charleszipp will work on a test for sure, wanted to get your initial thoughts on this first before going all too deep into keystrokes :) Will try to push them tonight/tomorrow my am. Thanks for the review!

File: TerraformCLI/src/tests/terraform-force-unlock/force-unlock_spec.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        new TestScenario(env.taskScenarioPath)
+            .assertExecutedTerraformCommand(env.expectedCommand)
+            .assertExecutedTerraformVersion()
+            // test runner does not expose env vars set within the task so cannot use this yet
+            //.andAssert((assertions) => new TaskExecutedWithEnvironmentVariables(assertions, expectedEnv));
+            .run();
+    });
+
+    it('azurerm with secure env file', function(){
+        const env = require('./force-unlock-azurerm-with-secure-env-file.env').env;
```
Comment (charleszipp) [inline TerraformCLI/src/tests/terraform-force-unlock/force-unlock_spec.ts:16]: @favoretti im not sure we need this test because the variables group is not enabled for `force-unlock` command.

File: TerraformCLI/src/terraform-force-unlock.ts [hunk 2]
```diff
@@ -41,14 +40,12 @@ export class TerraformForceUnlockHandler implements IHandleCommandString{
             command,
             tasks.getInput("workingDirectory"),
             tasks.getInput("environmentServiceName"),
-            tasks.getInput("commandOptions"),
+            tasks.getInput("lockID"),
```
Comment (charleszipp) [inline TerraformCLI/src/terraform-force-unlock.ts:43]: Should we disable the command options in `task.json` to prevent someone from trying to add the `-force` option or is there something else that will prevent that from being duplicated?

**Review by charleszipp — CHANGES_REQUESTED** [review]
Couple small changes then we should be good to go. Pipeline is passing for all other commands. Will add this command into the pipeline so it gets tested from there too.

File: TerraformCLI/src/tests/terraform-force-unlock/force-unlock_spec.ts (hunk 1 above)
Comment (favoretti) [reply TerraformCLI/src/tests/terraform-force-unlock/force-unlock_spec.ts:16]: Ah, that's true... Let me remove it.

File: TerraformCLI/src/terraform-force-unlock.ts (hunk 2 above)
Comment (favoretti) [reply TerraformCLI/src/terraform-force-unlock.ts:43]: Hmm, are you looking at an outdated version? I removed it, don't have it locally.

Comment (charleszipp) [reply TerraformCLI/src/terraform-force-unlock.ts:43]: Im still seeing it when adding the task from the pipeline. I think you would need to update the `commandOptions` input in task.json to enable only if command != force-unlock
image

Comment (charleszipp) [reply TerraformCLI/src/terraform-force-unlock.ts:43]: Note also not seeing the lock-id field show up either.

Comment (favoretti) [reply TerraformCLI/src/terraform-force-unlock.ts:43]: Hrm.. Let me double-check.

Comment (favoretti) [reply TerraformCLI/src/terraform-force-unlock.ts:43]: Ok, removed the announcement from overview, let's do a dark launch, I can test it on my pipelines once it works, I'll push a line into the readme.

**Review by charleszipp — APPROVED** [review]
